src/guides/v2.3/test/testing.md

```diff
@@ -236,15 +236,15 @@ MFTF tests are kept within its respective Module folder:
 
 <!-- Link Definitions -->
 [Magento definition of done]: {{ page.baseurl }}/contributor-guide/contributing_dod.html
-[Functional]: {{ page.baseurl }}/mtf/mtf_introduction.html
+[Functional]: {{ site.baseurl }}/mftf/docs/introduction.html
 [Web API Functional]: {{ page.baseurl }}/get-started/web-api-functional-testing.html
 [Integration]: {{ page.baseurl }}/test/integration/integration_test_execution.html
 [performance toolkit]: {{ page.baseurl }}/config-guide/cli/config-cli-subcommands-perf-data.html
 [JavaScript]: {{ page.baseurl }}/test/js/jasmine.html
 [PhpCs]: {{ page.baseurl }}/coding-standards/code-standard-php.html
 [PhpMd]: https://phpmd.org/
 [Magento backward compatibility policy]: {{ page.baseurl }}/contributor-guide/backward-compatible-development/
-[Functional Testing Framework]: {{ page.baseurl }}/mtf/mtf_introduction.html
+[Functional Testing Framework]: {{ site.baseurl }}/mftf/docs/introduction.html
```

Comment:
The paragraph where this link is used tells about an original framework. In this context, it may make sense to use a link to the archived repository: https://github.com/magento/mtf

Reply:
That paragraph should be rephrased to something like:

_The [Functional Testing Framework][] (FTF) was the original framework used for testing Magento. It used PHP scripts to run browser testing.
This framework is no longer supported and FTF tests were [ported over][] to the MFTF._

Reply:
Thanks! Fixed.